docs/template_plugin/src/template_executable_network.cpp

```diff
@@ -203,7 +203,7 @@ void TemplatePlugin::ExecutableNetwork::Export(std::ostream& modelStream) {
     // Note: custom ngraph extensions are not supported
     std::map<std::string, ngraph::OpSet> custom_opsets;
     std::stringstream xmlFile, binFile;
-    ngraph::pass::Serialize serializer(xmlFile, binFile, ngraph::pass::Serialize::Version::IR_V10, custom_opsets);
+    ngraph::pass::Serialize serializer(xmlFile, binFile, custom_opsets, ngraph::pass::Serialize::Version::IR_V11);
```

Comment:
I believe we should serialize as is - we information which is inside `ov::Function`?
What if IR v10 is passed?

Reply:
can we remove explicit `IR_V11` and serializer will always use latest version?